### PR TITLE
Add toggle button in replacement of upper and lower

### PR DESCRIPTION
**Pull request by Vishal1297**

### CODE REVIEW

File: src/components/TextForm.js [hunk 1]
```diff
@@ -24,14 +24,8 @@ export default function TextForm(props) {
...
     if (!text) return "";
-    let splitVal = text.trim().split(" ");
-    let string = "";
-    splitVal.forEach((elements) => {
-      if (elements) {
-        string += `${elements} `;
-      }
-    });
-    setText(string);
+    let txtWithoutTrailingLeadingSpaces = text.trim().split(" ").join(" ").trim();
```
Comment (Vishal1297) [inline src/components/TextForm.js:27]: while appending one extra space will be left, so removed using trim

File: src/components/TextForm.js [hunk 2]
```diff
@@ -44,6 +38,17 @@ export default function TextForm(props) {
     setText(lowerCaseText);
   };
 
+  /**
```
Comment (Vishal1297) [inline src/components/TextForm.js:41]: Will be used for single button for case toggling.

Comment (Vishal1297) [reply src/components/TextForm.js:41]: @Suyash99 

fixed

File: src/components/TextForm.js (hunk 1 above)
Comment (Vishal1297) [reply src/components/TextForm.js]: @Suyash99 

Fixed

**Comment by Suyash99** [comment]
@Vishal1297 Please push these changes on new Branch. Will pull and test these changes

**Comment by Vishal1297** [comment]
> @Vishal1297 Please push these changes on new Branch. Will pull and test these changes

Why new branch ?

**Comment by Vishal1297** [comment]
## Screenshot

image


### Done

- [x] Added Toggle
- [x] Fixed remove spaces

**Pull request closed without merging**